src/MessageHandler/DatasetSubmissionFilerHandler.php

```diff
@@ -102,7 +114,11 @@ public function __invoke(DatasetSubmissionFiler $datasetSubmissionFiler)
             // Log processing complete.
             $this->logger->info('Dataset submission process started', $loggingContext);
             foreach ($fileset->getFiles() as $file) {
-                $this->processFile($file, $loggingContext);
```

Comment:
I don't know if we need to shove this logging context around. It'll be the same information every log event. The only thing that's new in `processFile` it an generate it's own logging context there, it just need the file ID, the other information would we seen a few lines over it? Opinions? Do we need all the context for every log line?

Reply:
Maybe its not required, I think it would be safe to assume that those files are under the same submission. We can remove passing the logging context into the processFile method